### PR TITLE
Shapella upgrade. Improvements, test fixes, deps updates

### PR DESCRIPTION
**Pull request by karim-en**
## Contracts
- Update `near-sdk` to `4.1.1`
- Update `lighthouse` dependency to `aurora-3.5.1`
- Update BlockHeader to support [eip-4895](https://eips.ethereum.org/EIPS/eip-4895)
- Add Cappella config for `Goerli` network
* Remove `Ropsten` and `Kiln` networks
- Add Cappella config for `mainnet` network  https://github.com/ethereum/consensus-specs/blob/214760b59bc2355cf0259ae2d042634a0d830525/specs/capella/fork.md

## Eth2 crates

- Use `attested_beacon_header` instead of `active_header` to verify  sync committee update proof.  [specs](https://github.com/ethereum/consensus-specs/commit/c4dbd241acc93d29d82e06d18e0828a264c5bf29)
- Update relayer code to support new lodestar API version `1.5.1`, and Shapella hardfork.
- Request light client update by epoch for modified lodestar node  https://github.com/aurora-is-near/lodestar/commit/9b5f3ed1869d5a612d0ee5d49f8b646d985c5555

## Misc
- CI: utilize `git-lfs` for Ethereum, NEAR contracts matrices.
- Formatting fixes.

### CODE REVIEW

File: contracts/near/Cargo.toml [hunk 1]
```diff
@@ -19,3 +19,8 @@ overflow-checks = true
 [patch]
 [patch.crates-io]
 eth2_hashing = { path = "eth2_hashing" }
+eth2_ssz = { git = "https://github.com/aurora-is-near/lighthouse.git",  rev = "ec55f0b8eb997c80364f01afef74114ce139bcae"  }
+tree_hash = { git = "https://github.com/aurora-is-near/lighthouse.git",  rev = "ec55f0b8eb997c80364f01afef74114ce139bcae"}
+tree_hash_derive = { git = "https://github.com/aurora-is-near/lighthouse.git",  rev = "ec55f0b8eb997c80364f01afef74114ce139bcae"}
+eth2_serde_utils = { git = "https://github.com/aurora-is-near/lighthouse.git",  rev = "ec55f0b8eb997c80364f01afef74114ce139bcae" }
+eth2_ssz_derive = { git = "https://github.com/aurora-is-near/lighthouse.git",  rev = "ec55f0b8eb997c80364f01afef74114ce139bcae" }
```
Comment (sept-en) [inline contracts/near/Cargo.toml:26]: Please use a tag instead.

File: contracts/near/eth-client/Cargo.toml [hunk 2]
```diff
@@ -10,14 +10,12 @@ crate-type = ["cdylib", "rlib"]
...
+near-sdk = "4.1.1"
 borsh = "0.9.3"
-wee_alloc = { version = "0.4.5", default-features = false, features = [] }
-rlp = "0.4.2"
-futures = "0.1.26"
+rlp = "0.5.2"
 primal = "0.2.3"
 arrutil = "0.1.2"
-ethash = { git = "https://github.com/nearprotocol/rust-ethash" }
+ethash = { git = "https://github.com/aurora-is-near/rust-ethash", rev = "a27b2c7217f74c4470d7cd0b26b41b58431a26ee", default-features = false }
```
Comment (sept-en) [inline contracts/near/eth-client/Cargo.toml:18]: Please use a tag instead.

**Review by sept-en — CHANGES_REQUESTED** [review]

**Review by olga24912 — APPROVED** [review]

File: contracts/near/Cargo.toml (hunk 1 above)
Comment (karim-en) [reply contracts/near/Cargo.toml]: Fixed 3a57c023e2ff75b75f5fbeb2c1c97c15531a5fd7

File: contracts/near/eth-client/Cargo.toml (hunk 2 above)
Comment (karim-en) [reply contracts/near/eth-client/Cargo.toml]: Fixed 7ff9642f1500a3cd69babe8a9a44fb6760933634

**Review by sept-en — CHANGES_REQUESTED** [review]

**Review by sept-en — APPROVED** [review]

**Pull request merged**